README.md

```diff
@@ -30,36 +30,96 @@ and a beautiful sign in button appears as shown below.
 ```dart
 import 'package:aad_b2c_webview/src/login_azure.dart';
 
-class MyLoginPage extends StatelessWidget {
-  const MyLoginPage({Key? key}) : super(key: key);
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key? key}) : super(key: key);
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  String? jwtToken;
+  String? refreshToken;
 
   @override
   Widget build(BuildContext context) {
+    const aadB2CClientID = "<clientId>";
+    const aadB2CRedirectURL = "<azure_active_directory_url_redirect>";
+    const aadB2CUserFlowName = "B2C_<name_of_userflow>";
+    const aadB2CScopes = ['openid', 'offline_access'];
+    const aadB2CUserAuthFlow =
+        "https://<tenant-name>.b2clogin.com/<tenant-name>.onmicrosoft.com"; // https://login.microsoftonline.com/<azureTenantId>/oauth2/v2.0/token/
+    const aadB2TenantName = "<tenant-name>";
+
     return Scaffold(
-      body: AADLoginButton(
-        userFlowUrl:
-        '<https://<tenant_id>.b2clogin.com/<tenant_id>.onmicrosoft.com/oauth2/v2.0/authorize>',
-        clientId: '<client-id>',
-        userFlowName: 'B2C_<Name_of_UserFlow>',
-        redirectUrl: '<redirect_url>',
-        onRedirect: (BuildContext context){
-          ///Handle navigation to whatever page you choose
-          ///Use the build context for navigation
-        },
-        context: context,
-        onAccessToken: (value) {
-          ///Store or use access token from here
-        },
-        onIDToken: (value) {
-          ///Store or use ID token from here
-        },
-        onRefreshToken: (value) {
-          ///Store or use refresh token from here
-        },
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            /// Login flow
+            AADLoginButton(
+              userFlowUrl: aadB2CUserAuthFlow,
+              clientId: aadB2CClientID,
+              userFlowName: aadB2CUserFlowName,
+              redirectUrl: aadB2CRedirectURL,
+              context: context,
+              scopes: aadB2CScopes,
+              onAnyTokenRetrieved: (Token anyToken) {
+                if (kDebugMode) {
+                  print(
+                      "Any token of type: ${anyToken.type.name}: ${anyToken.value}");
+                }
+              },
+              onIDToken: (Token token) {
+                jwtToken = token.value;
+                if (kDebugMode) {
+                  print("Id Token: ${token.value}");
```

Comment:
As tokens will be printed in plain text, we can  avoid the print statements and securely store the token values.

Reply:
Lovely! Iw ill work on it! and soon push a new update :)

Reply:
@mohanajuhi166  done! :)